modules/statics/src/coins.ts

```diff
@@ -419,6 +419,7 @@ export const coins = CoinMap.fromCoins([
     '0xd829664cdbf3195b2ce76047a65de29e7ed0a9a8',
     UnderlyingAsset.ALTBULL
   ),
+  erc20('amkt', 'Alongside Index', 18, '0xf17a3fe536f8f7847f1385ec1bc967b2ca9cae8d', UnderlyingAsset.AMKT),
```

Comment:
Double checking, When I checked the contract name it is `Alongside Crypto Market Index` shouldn't we use the same name?

Reply:
@sachushaji thanks for catching this. It's changed.